index.php

```diff
@@ -197,6 +197,7 @@
 <footer class="footer">
     <p>
         <a href="http://www.qiche8.net/" target="_blank"><img src="/assets/images/static/qiche8.jpg" class="img-responsive img-rounded img-thumbnail" alt="远人部落" title="远人部落" width="70" height="70"></a>
+        <a href="https://www.specialized.com/cn/zh" target="_blank"><img src="/assets/images/static/specialized.png" class="img-responsive img-rounded img-thumbnail" alt="闪电" title="闪电" width="70" height="70"></a>
```

Comment:
Use an external image is not a good idea cause the external dependency may change its behavior. I will include it in the corresponding directory.